A33-Fault-Injection.md

```diff
@@ -0,0 +1,269 @@
+Client-Side Fault Injection
+----
+* Author(s): lidiz
+* Approver: roth, ejona, dfawley
+* Status: In Review
+* Implemented in: all languages
+* Last updated: August 13th, 2020
+* Discussion at: https://groups.google.com/g/grpc-io/c/d9dI5Hy9zzk
+
+## Abstract
+
+Fault injection is an essential feature that assists developers in building fault-tolerant applications. To keep gRPC’s competitive advantage, this doc aims to design a fault injection mechanism on the client side for gRPC while allowing transparent transition from Envoy configuration.
+
+## Background
+
+### Envoy HTTPFilter
```

Comment:
I think this section should just be replaced with a reference to #219.  We should explicitly state that the FI functionality depends on xDS HTTP filter support.

Reply:
Removed most descriptions and linked to A39.